lib/getTeam.ts

```diff
@@ -0,0 +1,66 @@
+import { Team } from "@prisma/client";
+import { GetServerSidePropsContext } from "next";
+import prisma from "./prisma";
+
+export const getTeam = async (context: GetServerSidePropsContext): Promise<Team | null> => {
+  let teamIdOrSlug = null;
+  let team = null;
+
+  switch (process.env.NODE_ENV) {
```

Comment:
Having environment related logic is a recipe for unexpected side-effects when changing environments (because it can introduce code that only runs in specific environments), this is why it is considered bad practise to be avoided at all cost.